bootstraptest/test_yjit_new_backend.rb

```diff
@@ -144,6 +144,20 @@ def foo()
     foo()
 }
 
+# opt_send_without_block (VM_METHOD_TYPE_CFUNC)
+assert_equal '["nil", 4, "aba", true, ["a", "b"]]', %q{
+    def foo
+      [
+        nil.inspect,               # 0
+        2.pow(2),                  # 1
+        "abc".tr("c", "a"),        # 2
+        respond_to?(:inspect),     # -1
+        "a\nb".lines(chomp: true), # -1
+      ]
+    end
+    foo
+}
+
```

Comment:
It's best to split these into separate tests, because otherwise YJIT can side-exit in the middle of the tests and only run some of the code you would expect it to run.

Reply:
That makes sense. I split the tests here https://github.com/Shopify/ruby/pull/357/commits/b9f70df0c664f4c8756ae6e36701af55538a48d3.

> Everything else looks good

Thank you for your review! Merging.